examples/main.cpp

```diff
@@ -131,9 +131,9 @@ void sample2_mlp(const string& data_dir) {
 #if defined(_MSC_VER) && _MSC_VER < 1800
   // initializer-list is not supported
   int num_units[] = {28 * 28, num_hidden_units, 10};
-  auto nn         = make_mlp<tan_h>(num_units, num_units + 3);
+  auto nn         = make_mlp<tanh_layer>(num_units, num_units + 3);
```

Comment:
@edgarriba I think we need to drop backward compatibility. Anyway that's gonna happen (once again) on switch to `Tensor`. BTW, this means we better drop back to 0.*.* if we adopt https://github.com/tiny-dnn/tiny-dnn/issues/545